### DISCUSSION TITLE
Slider reverse arrows do not match stable behavior

### DISCUSSION THREAD

**Discussion by nobully-osu** [ppy/osu#25986] [Ideas] [answered]
Wasn't sure if this was intentional or not or a bug, but in lazer the slider reverse arrows are different in animation behavior than stable. In lazer, they fade in slower and disappear faster, and grow instantly.

https://github.com/ppy/osu/assets/30556038/49f96b32-5201-4cf4-a751-da9f8dc680b4

If this is intentional or not going to be changed, it's 100% something I can live with and I'm sure not even that many people noticed. I prefer the faster disappearing part as well. But, there has been a large focus on bringing most everything in parity with stable so I thought I would still share.

**Comment by peppy** [comment]
might be dimming

**Reply by nobully-osu** [reply]
Perhaps, I'll try to investigate more.

**Comment by peppy** [comment, marked as answer]
The animation definitely shouldn't be resetting though, which it looks to be currently.

https://github.com/ppy/osu/issues/26566
